MD/md-6/README.md

```diff
@@ -0,0 +1,80 @@
+
+  1. Title: A concise name for the desideratum.
+  2. User Journey: A one or two-sentence statement focusing on the "user" (could be a human, machine, software, etc.) and their interaction or experience.
+  3. Description (optional): A more detailed explanation if needed.
+  4. Justification: The reasoning behind the desideratum. Why is it necessary or desired?
+  5. Recommendations (optional): Suggestions or guidance related to the desideratum.
+
+  Format as:
+
+  ### Desideratum Title
+
+  **User Journey**: [user] can [action].
+
+  **Description**: <More detailed explanation if needed (optional)>
+
+  **Justification**: <Why this is a significant or required desideratum>
+
+  **Recommendations**: <Any specific guidance or suggestions (optional)>
+
+  TODO: Remove this comment before finalizing.
+-->
+### D1: Model Transaction Fairness in Suzuka Network
+**User Journey**: A researcher or protocol implementer can understand what transaction fairness means in the Suzuka Network.
+
+**Justification**: Transaction fairness is a key component of any blockchain network. It is important to understand what it means in the context of the Suzuka Network.
+
+**Recommendations**:
+- The strongest form of formally defined fairness is currently [batch-order-fairness](https://link.springer.com/chapter/10.1007/978-3-030-56877-1_16). What would it mean for the Suzuka Network to achieve this. 
+- Transaction fairness should consider:
+    - Time in flight
+    - Congestion
+    - Ingress node
+    - Causality
+
+### D2: Accept Only Blocks Signed by Staked Validators
+**User Journey**: A staked Suzuka Validator is the only entity that can submit blocks to the network which will later be processed by the Aptos executor.
+
+**Justification**: This will prevent DOS attacks on the network by requiring a stake to submit blocks.
+
+**Recommendations**:
+- Be sure to consider what is lost in this approach. For example, currently, a developer could bypass a Suzuka Full Node an submit directly to the DA--allowing them to avoid any potential censorship. This would be lost in a naive implementation of this approach, as they would be required to either (a) submit via a full-node or (b) stake themselves.
+
+### D3: Invalidation or Validator-slashing for Producing Unfair Blocks
+**User Journey**: Validators can be slashed for producing unfair blocks.
+
+**Justification**: This will disincentivize validators from producing unfair blocks.
+
+**Recommendations**:
+- This would be a highly experimental feature and should be researched and implemented with caution. 
+- Deterministic transaction and block ordering should cover most of the cases of unfairness. These are discussed in [MD-7](../md-7/README.md). However, in theory you could also slash on conditions like including too many transactions from one user, sending too many blocks at a given time, sending too few blocks at a given time, sending blocks of unequal size when transactions are available, etc.
```

Comment:
I would argue it only covers a sub category rather than the majority. 

> However, in theory you could also slash on conditions like ...

i am not sure why we need to slash. could we have a rule on the executor side whether to accept a batch or not based on these rules instead? or does the DA remove the batch wrapping? where would the slashing be enforced?

Reply:
In the case you have the staking requirement mentioned above, would always slash in the settlement contract.